Pokedex_iosTests/Pokedex_iosTests.swift

```diff
@@ -8,7 +8,27 @@
 import XCTest
 @testable import Pokedex_ios
 
+let data = PokemonRaw(id: 4, name: "some", generation: Generation(id: 4, name: "gen1"), pokemonDetails: [PokemonDetail(name: "detail1", height: 5, weight: 5, types: [TypeElement(type: Generation(id: 3, name: "gen1"))])])
```

Comment:
I'd be better to set this data in a different file inside Test folder, so other unit test can use it